yarn.lock

```diff
@@ -28184,6 +28292,35 @@ webpack@^4.43.0:
     watchpack "^1.7.4"
     webpack-sources "^1.4.1"
 
+webpack@^5.21.2:
```

Comment:
I hate yarn...this should NOT be a separate entry. I'll push a change fixing this (because I don't know a concise explanation of how to do it).